### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'load'

### ISSUE THREAD

**Issue by ramonskie** [jrnewell/spotify-ripper#11]
sometimes when i try to rip a playlist it shows the following
the strange thing is that when i kill the process and start it again it
continuous 

```
spotify-ripper/priscilla/Dolly Parton - Jolene.mp3
Progress: [======================================= ] 2:40 / 2:41
Rip complete
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "./ripper.py", line 143, in run
    self.set_id3_and_cover(track)
  File "./ripper.py", line 383, in set_id3_and_cover
    image.load()
AttributeError: 'NoneType' object has no attribute 'load'
```

**Comment by jrnewell** [comment]
Fixed in commit aabc961.

**Comment by ramonskie** [comment]
:+1:  for the fast response
and it works
